Uno/NuGetPackageExplorer.Shared/Legacy/Views/Behaviors/TreeViewBehavior.cs

```diff
@@ -195,35 +207,18 @@ void ToggleExpansion(TreeView sender, TreeViewItemInvokedEventArgs args)
             }
         }
 
-        [SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "managed via DoubleClickCommandDisposable")]
-        private static void OnDoubleClickCommandChanged(TreeView control, DependencyPropertyChangedEventArgs e)
-        {
-            GetDoubleClickCommandDisposable(control)?.Dispose();
-            if (e.NewValue is ICommand command)
-            {
-                var subscription = Observable
-                    .FromEventPattern<DoubleTappedEventHandler, DoubleTappedRoutedEventArgs>(
-                        h => control.DoubleTapped += h,
-                        h => control.DoubleTapped -= h
-                    )
-                    .Select(x => x.EventArgs.OriginalSource)
-                    .Subscribe(x =>
-                    {
-                        try
-                        {
-                            if (command.CanExecute(x))
-                            {
-                                command.Execute(x);
-                            }
-                        }
-                        catch (Exception ex)
-                        {
-                            typeof(TreeViewBehavior).Log().Error("failed to execute DoubleClick command: ", ex);
-                        }
-                    });
+        #endregion
+        #region DependencyProperty: AutoToggleItemExpansionDisposable
 
-                SetDoubleClickCommandDisposable(control, subscription);
-            }
-        }
```

Comment:
It's best to avoid cosmetic changes along with behavior changes, it makes reviewing more difficult.